crates/sui-source-validation-service/src/lib.rs

```diff
@@ -99,9 +100,10 @@ impl CloneCommand {
         // Args to clone empty repository
         let cmd_args: Vec<OsString> = vec![
             ostr!("clone"),
-            ostr!("-n"),
-            ostr!("--depth=1"),
+            ostr!("--no-checkout"),
+            ostr!("--depth=1"), // implies --single-branch
```

Comment:
Just some cosmetic changes for the clone command, to make it a bit more clear. In some cases you'd need to specify `--single-branch` when using `--branch`, but `--depth` implies this.